contrib/kakoune/patches/fix-issue-63281-llvm-llvm-project.patch

```diff
@@ -0,0 +1,137 @@
+diff --git a/src/command_manager.cc b/src/command_manager.cc
```

Comment:
where is this from? did you upstream it?

Reply:
ah, i see it's https://github.com/mawww/kakoune/commit/7577fa1b668ea81eb9b7b9af690a4161187129dd . please use the actual .patch from the url when using those at least, otherwise there is no way to know :p

Reply:
i made it myself. there are some differences in it in terms of line numbers

Reply:
if diff is capable of dealing with that though, i wasn't aware. do let me know and i'll use the patch directly from there

Reply:
> if diff is capable of dealing with that though,

it is